utils.ts

```diff
@@ -4,14 +4,10 @@ import bugsnag from '@bugsnag/js'
 import { fromCoordinates } from '@conveyal/lonlat'
 import { getDistance } from 'geolib'
 import fetch from 'node-fetch'
-
 import type { LonLatOutput } from '@conveyal/lonlat'
 import type { Feature, FeatureCollection, Position } from 'geojson'
-import type {
-  AutocompleteQuery,
-  ReverseQuery,
-  SearchQuery
-} from '@opentripplanner/geocoder'
+import { RedisClientType } from 'redis'
+import { AnyGeocoderQuery } from '@opentripplanner/geocoder/lib/geocoders/types'
```

Comment:
`import type`
I don't think this makes any difference in practice but it is a bit more descriptive.